server/backend/users/tests/test_models.py

```
@@ -0,0 +1,69 @@
+import pytest
+from users.models import Instructor, Student, User
```

Ruff (0.8.2)

`users.models.Student` imported but unused

Remove unused import: `users.models.Student`

(F401)